plugins/utils/which-key.nix

```diff
@@ -55,257 +265,308 @@ with lib;
       ];
     };
 
+    notify = defaultNullOpts.mkBool true "Show a warning when issues were detected with your mappings.";
+
+    triggers = defaultNullOpts.mkListOf (with types; attrsOf anything) [
+      {
+        __unkeyed = "<auto>";
+        mode = "nxsot";
+      }
+    ] "Manually setup triggers";
+
+    defer =
+      defaultNullOpts.mkLuaFn
+        (mkRaw ''
+          function(ctx)
+            return ctx.mode == "V" or ctx.mode == "<C-V>
+            end'')
+        ''
+          Start hidden and wait for a key to be pressed before showing the popup.
+
+          Only used by enabled xo mapping modes.
+        '';
+
     plugins = {
-      marks = helpers.defaultNullOpts.mkBool true "shows a list of your marks on ' and `";
+      marks = defaultNullOpts.mkBool true "Shows a list of your marks on ' and `";
```

Comment:
We need to format descriptions as markdown, also we prefer to have punctation:

```suggestion
      marks = defaultNullOpts.mkBool true "Shows a list of your marks on `'` and <code>\`</code>.";
```

---

Also, I wonder if many of these would be more readable with indented strings?

```suggestion
      marks = defaultNullOpts.mkBool true ''
        Shows a list of your marks on `'` and <code>\`</code>.
      '';
```

Reply:
I'm not sure if we need <code>&lt;code&gt;\\\`&lt;/code&gt;</code> or if <code>&lt;code&gt;\`&lt;/code&gt;</code> would be enough :thinking: 

Based on github's markdown, we'd need the backslash, but commonmark may be different.

This can be double-checked by entering a devshell and running `serve-docs`

Reply:
![image](https://github.com/user-attachments/assets/5e206ae2-5e14-406e-8d9c-40559cf6a1f8)

Reply:
Maybe the doc's implementation blocks HTML elements in the markdown? :sob: 

We can try <code>\`\\\`\`</code>, not sure if it'll work :face_with_diagonal_mouth:

Reply:
I'm trying out stuff locally, didn't realize we had the serve-docs command. That's really helpful!

Reply:
Looks like 
```
`` ` ``
```

Reply:
This may help: https://meta.stackexchange.com/a/70679

Depending on the markdown flavour we're dealing with, we may be able to do <code>\`\`some code span with a \` in it\`\`</code>

Or alternatively, we may be able to use the html code `&#96;` as <code>\`&amp;#96;\`</code>

Reply:
Yeah, that looks like what I ended up doing. Just wrapping it in a double backtick